test/template/www/js/scenarioRestart.js

```diff
@@ -36,7 +36,8 @@ var app = {
         window.codePush.restartApplication(
             function () {
                 /* success */
-                app.sendTestMessage("RESTART_SUCCEEDED");
+                // does not always send...rely on RESTART_FAILED to provide failure state instead of checking for RESTART_SUCCEEDED
+                // app.sendTestMessage("RESTART_SUCCEEDED");
```

Comment:
But I'm ok with merging this change if it's easier. I guess its not super important. I'll leave it up to you.